### DISCUSSION TITLE
OracleDB 사용을 위한 접속 가이드 필요

### DISCUSSION THREAD

**Discussion by laigasus** [cloud2scape/.github#21] [on-일반]
개발 수행을 위한 oracle 컨테이너가 dockerhub 에는 없습니다.
제가 임시로 사용할 수 있도록 방안이 필요합니다.

별도 oracle 관련 프로그램 설치작업 없이 url 만으로만 접속할 수 있는 방법이 필요합니다.
이에 mysql 처럼 만으로 접속할 수 있는 방법을 제시했으면 합니다.
  - host
  - username
  - password
  - database 

당장은 mysql 로 작업하고
조치가 이루어지는 대로 oracle 적용하겠습니다.

![image](https://github.com/user-attachments/assets/ded21e13-c4b3-470a-b1c6-bad8b9919a17)

**Comment by gimsoomin** [comment]
확인했습니다

**Comment by laigasus** [comment]
DB는 mysql-mariadb 계열을 사용하기로 결정되었음
